SU-SWS/decanter
# D8CORE-2630: Updating the alert basic link hover colors & underline

# READY FOR REVIEW

# Summary
- We need to change the link colors on hover/focus states for our alerts to meet accessibilty.

# Needed By (Date)
- Sept 3

# Urgency
- High

# Steps to Test

1. Checkout this branch
2. Try out it out in the developer/ playground.
3. Look at each alert variant which can be found at /patterns/alert and make sure that the links' hover and focus only show an underline and do not change the text color. Verify the Info alert links are not just underlined in normal state.

# Affected Projects or Products
- Stanford Basic
- SOE
- Adapt (I think)

# Associated Issues and/or People
- D8CORE-2630
- @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
